Neos.Flow/Classes/I18n/Xliff/XliffParser.php

```diff
@@ -28,67 +26,26 @@
  * @throws Exception\InvalidXliffDataException
  * @see http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html [1]
  * @see http://docs.oasis-open.org/xliff/v1.2/xliff-profile-po/xliff-profile-po-1.2-cd02.html#s.detailed_mapping.tu [2]
+ * @deprecated use \Neos\Flow\I18n\Xliff\V12\XliffParser instead. Will be removed in Flow 5.0
+ * Also consider using
+ * @see \Neos\Flow\I18n\Xliff\Service\XliffFileProvider
+ * as it is capable of label overrides
  */
-class XliffParser extends AbstractXmlParser
+class XliffParser extends V12\XliffParser
 {
     /**
      * Returns array representation of XLIFF data, starting from a root node.
      *
      * @param \SimpleXMLElement $root A root node
      * @return array An array representing parsed XLIFF
      * @throws Exception\InvalidXliffDataException
-     * @todo Support "approved" attribute
+     * @deprecated use \Neos\Flow\I18n\Xliff\V12\XliffParser::doParsingFromRoot instead
```

Comment:
Same here, the class has no `@api` flag, the method has nop `@api`flag and is protected, so in theory we can adjust is as we like. So, yes, this is fine.